test/non-serializable.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+
+const metaschema = require('metaschema');
+const metatests = require('metatests');
+
+const { SchemaValidationError, MetaschemaError } = metaschema.errors;
+const { options, config } = require('../lib/metaschema-config/config');
+
+metatests.test('Non-serializable functions', async test => {
+  try {
+    await metaschema.fs.load('test/fixtures/non-serializable', options, config);
+    test.fail();
```

Comment:
IIRC, `test.fail()` requires a message. Also, you can replace it with `test.rejects()` and remove `try/catch` with the next `metatests` release.